ridlbe/ccmx11/facets/dds/visitorbase.rb

```diff
@@ -16,9 +16,27 @@ module Cxx11
     #
     class VisitorBase
 
-      # Need to put more native here, shouldn't add "DDS_Native" to templates
-      def native_type_name
-        scoped_cxxtype.start_with?('::') ? "DDS_Native#{scoped_cxxtype}" : "#{scoped_cxxtype}"
+      # For all implied C++11 types we need the escaped C++ namespace but the unescaped C++ name
```

Comment:
This comment is not clear. There seems to miss a 'not' somewhere. What is that needs to be used the escaped or unescaped name?